app/models/manageiq/providers/openstack/cloud_manager/host_aggregate/operations.rb

```diff
@@ -0,0 +1,37 @@
+module ManageIQ::Providers::Openstack::CloudManager::HostAggregate::Operations
+  include SupportsFeatureMixin
```

Comment:
this should be like this, because you want those `supports` calls on HostAggregate
```ruby
extend ActiveSupport::Concern
included do
  supports :add_host
  supports :remove_host
end
```

@Fryguy this could be avoided by safeguarding as I proposed here https://github.com/ManageIQ/manageiq/pull/11322